### DISCUSSION TITLE
jsonata query box

### DISCUSSION THREAD

**Discussion by yosiasz** [grafana/grafana-infinity-datasource#472] [Ideas]
Greetings

Not sure if this is a missing feature or I don't know how to use this feature if it exists.
Would be nice if the UQL query box allowed CR+LF for easy viewing of code. Currently it seems you need to have your code in one long line. This is not formatting friendly for Functional Programming a lot of stuff?

https://try.jsonata.org/GIZGT873L

Thanks!

**Comment by yesoreyeram** [comment]
yea I understand the pain... But I need to find a fix for parsing multi line string in [UQL](https://github.com/yesoreyeram/uql). (

**Reply by yosiasz** [reply]
thanks @yesoreyeram low priority request. works great with https://tools.knowledgewalls.com/online-multiline-to-single-line-converter. thanks!

**Reply by gdiana** [reply]
Very useful tool @yosiasz
